tests/unit/amo/sagas/testLanding.js

```diff
@@ -0,0 +1,123 @@
+import { hideLoading, showLoading } from 'react-redux-loading-bar';
+import SagaTester from 'redux-saga-tester';
+
+import * as api from 'core/api';
+import { getLanding, loadLanding } from 'amo/actions/landing';
+import { LANDING_PAGE_ADDON_COUNT } from 'amo/constants';
+import landingReducer from 'amo/reducers/landing';
+import landingSaga from 'amo/sagas/landing';
+import {
+  ADDON_TYPE_EXTENSION,
+  LANDING_LOADED,
+  SEARCH_SORT_POPULAR,
+  SEARCH_SORT_TOP_RATED,
+} from 'core/constants';
+import { ErrorHandler } from 'core/errorHandler';
+import apiReducer from 'core/reducers/api';
+import {
+  createAddonsApiResult, dispatchSignInActions, fakeAddon,
+} from 'tests/unit/amo/helpers';
+
+describe('amo/sagas/landing', () => {
+  describe('fetchLandingAddons', () => {
+    let apiState;
+    let errorHandler;
+    let mockApi;
+    let sagaTester;
+
+    beforeEach(() => {
+      errorHandler = new ErrorHandler({
+        id: 'some-error-handler',
+        dispatch: sinon.stub(),
+      });
+      mockApi = sinon.mock(api);
+
+      const { state } = dispatchSignInActions();
+      apiState = state.api;
+      sagaTester = new SagaTester({
+        initialState: state,
+        reducers: { landing: landingReducer, api: apiReducer },
+      });
+
+      sagaTester.start(landingSaga);
+    });
+
+    function _getLanding(overrides = {}) {
```

Comment:
This seems to be a common pattern in our tests, wonder if we could write a helper for it... or if it's not needed.